packages/config/src/options.ts

```diff
@@ -435,7 +435,7 @@ const runtimeOptions: Array<RuntimeConfigOption> = [
     // we may not be able to get the value to ignore.
     name: 'additionalIgnorePattern',
     defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? defaultSpecPattern.e2e : undefined,
-    validation: validate.isString,
+    validation: validate.isNullOrStringOrArrayOfStrings,
```

Comment:
Not sure what is the most common pattern in this file, but instead of allowing `undefined`, we could simply default to `[]`.

```js
    defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? defaultSpecPattern.e2e ?? [] : [],
``

Reply:
That looks so much better, thanks!